x-pack/plugins/apm/public/components/shared/charts/CustomPlot/Legends.js

```diff
@@ -66,16 +66,16 @@ export default function Legends({
 
   return (
     <Container>
-      {series.map((serie, i) => {
+      {series.map((series, i) => {
```

Comment:
I appreciate the effort to fix this spelling error, but I don't feel good about changing `serie` to `series`. In this instance the singular `series` will start shadowing the plural `series`, which can cause problems.
Much appreciated if separate words describing plural and singular series can be found - but that's probably better for another PR.

Reply:
Will drop from this PR